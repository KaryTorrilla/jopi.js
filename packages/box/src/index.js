import styled from 'styled-components'
import {
  space,
  color,
  layout,
  flexbox,
  border,
  typography,
  position,
  variant,
} from 'styled-system'
import css from '@styled-system/css'

const sx = props => css(props.sx)(props.theme)
const base = props => css(props.__css)(props.theme)
const theme = ({ tx = 'variants' }) =>
  variant({
    scale: tx,
  })

export const Box = styled.div(
  {
    boxSizing: 'border-box',
    margin: 0,
    minWidth: 0,
  },
  base,
  theme,
  sx,
  space,
  color,
  layout,
  flexbox,
  typography,
  border,
  position
)

export const Flex = styled(Box)({
  display: 'flex',
})
