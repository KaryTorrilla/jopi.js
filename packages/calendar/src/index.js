import React from 'react'
import FullCalendar from '@fullcalendar/react' // must go before plugins
import dayGridPlugin from '@fullcalendar/daygrid' // a plugin!
import '@oneloop/theme/styles/globals.css'
import '../styles/styles.css'

export const Calendar = () => {
  /*   const calendarOptions = {
      events: [],
      plugins: [timeGridPlugin],
      initialView: 'timeGridDay',
      nowIndicator: true,
      headerToolbar: {
        start: 'prev',
        center: 'title',
        end: 'next',
      },
      editable: true,
      eventResizableFromStart: true,
      dragScroll: true,
      dayMaxEvents: 2,
      navLinks: true,
    } */

  return (
    <div className='calendarContainer'>
      <FullCalendar
        plugins={[dayGridPlugin]}
        initialView="dayGridMonth"
      />
    </div>
  )
}
