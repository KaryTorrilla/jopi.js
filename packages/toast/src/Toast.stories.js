import React from 'react'
import { useToggle } from '@oneloop/hooks'
import { Button } from '@oneloop/button'

import { Toast } from '.'

export default {
  component: Toast,
  title: 'Toast',
}
export const base = () => <Toast type='base' variant='toastBase' text="Seleccione una propiedad"/>
     

export const normal = () =>
  React.createElement(() => {
    const [openAlert, setOpenAlert] = useToggle(false)
    return (
      <React.Fragment>
        <Button onClick={setOpenAlert}>Show Toast</Button>
        <br />
        <br />
        {openAlert && (
          <Toast type='normal' variant='toastNormal' text="Seleccione una propiedad" closeFunction={setOpenAlert} />
        )}
      </React.Fragment>
    )
  })

export const alert = () =>
  React.createElement(() => {
    const [openAlert, setOpenAlert] = useToggle(false)
    return (
      <React.Fragment>
        <Button onClick={setOpenAlert}>Show alert</Button>
        <br />
        <br />
        {openAlert && (
          <Toast variant='toastAlert' type="alert" text="Alert message" closeFunction={setOpenAlert} />
        )}
      </React.Fragment>
    )
  })

export const check = () =>
  React.createElement(() => {
    const [openAlert, setOpenAlert] = useToggle(false)
    return (
      <React.Fragment>
        <Button onClick={setOpenAlert}>Show alert</Button>
        <br />
        <br />
        {openAlert && (
          <Toast variant='toastCheck' type="check" text="Success message" closeFunction={setOpenAlert} />
        )}
      </React.Fragment>
    )
  })

  export const error = () =>
  React.createElement(() => {
    const [openAlert, setOpenAlert] = useToggle(false)
    return (
      <React.Fragment>
        <Button onClick={setOpenAlert}>Show alert</Button>
        <br />
        <br />
        {openAlert && (
          <Toast variant='toastError' type="error" text="Error message" closeFunction={setOpenAlert} />
        )}
      </React.Fragment>
    )
  })
  export const loading = () =>
  React.createElement(() => {
    const [openAlert, setOpenAlert] = useToggle(false)
    return (
      <React.Fragment>
        <Button onClick={setOpenAlert}>Show alert</Button>
        <br />
        <br />
        {openAlert && (
          <Toast variant='toastNormal' type="loading" text="Cargando..." closeFunction={setOpenAlert} />
        )}
      </React.Fragment>
    )
  })
  export const exporting = () =>
  React.createElement(() => {
    const [openAlert, setOpenAlert] = useToggle(false)
    return (
      <React.Fragment>
        <Button onClick={setOpenAlert}>Show alert</Button>
        <br />
        <br />
        {openAlert && (
          <Toast variant='toastNormal' type="loading" text="Exportando..." closeFunction={setOpenAlert} />
        )}
      </React.Fragment>
    )
  })
  export const success = () =>
  React.createElement(() => {
    const [openAlert, setOpenAlert] = useToggle(false)
    return (
      <React.Fragment>
        <Button onClick={setOpenAlert}>Show alert</Button>
        <br />
        <br />
        {openAlert && (
          <Toast variant='toastSuccess' type="success" text="Report export ready!" closeFunction={setOpenAlert} />
        )}
      </React.Fragment>
    )
  })

