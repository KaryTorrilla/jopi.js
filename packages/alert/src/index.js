import React from 'react'
import { Flex } from '@oneloop/box'
import { Check, Exclamation, Times } from '@oneloop/icons'
import { Text } from '@oneloop/text'
import { Button } from '@oneloop/button'

export const Alert = ({ type = 'success', text, ...props }) => (
  <Flex
    variant="alert"
    {...props}
    __css={{
      width: '100%',
      bg: 'white',
      borderRadius: 'circle',
      boxShadow: 'active',
      justifyContent: 'space-between',
      p: 1,
    }}
  >
    {type === 'success' && (
      <Check
        bg="success"
        color="white"
        borderRadius="circle"
        size={40}
        p={3}
        mr={5}
      />
    )}
    {type === 'warning' && (
      <Exclamation
        bg="warning"
        color="white"
        borderRadius="circle"
        size={40}
        p={3}
        mr={5}
      />
    )}
    {type === 'danger' && (
      <Times
        bg="danger"
        color="white"
        borderRadius="circle"
        size={40}
        p={3}
        mr={5}
      />
    )}
    <Text variant="body.0" display="flex" alignItems="center" flexGrow={1}>
      {text}
    </Text>
    <Button
      size={32}
      bg="#F6F2F2"
      p={3}
      display="inline-flex"
      alignSelf="center"
      mr={2}
    >
      <Times color="#716F6F" m="auto" />
    </Button>
  </Flex>
)
