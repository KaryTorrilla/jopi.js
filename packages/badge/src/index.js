import React from 'react'
import { Box } from '@oneloop/box'
import '@oneloop/fonts'

export const Badge = ({ isMedium, hasIcon, isNotButton, ...props }) => (
  <Box
    as={ isNotButton ? "" : "button" }
    tx="buttons"
    variant="primary"
    {...props}
    __css={{
      lineHeight: '14px',
      height: isMedium ? '22px' : '16px',
      display: 'inline-block',
      fontSize: '10px',
      fontFamily: 'primary',
      textAlign: 'center',
      textTransform: 'uppercase',
      fontWeight: 'bold',
      letterSpacing: '0.05em',
      px: 2,
      py: '2px',
      color: 'white',
      bg: 'primary',
      border: 0,
      borderRadius: isMedium ? '14px' : '8px',
      flexDirection: 'row',
      justifyContent: 'center',
      alignItems: 'center',
      padding: hasIcon ? '4px 8px' : '1px 5px',
      position: 'static',
      flex: 'none',
      order: 0,
      flexGrow: 0,
      margin: '0px 0px',
    }}
  />
)
