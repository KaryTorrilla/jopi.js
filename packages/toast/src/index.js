import React from 'react'
import { Flex, Box } from '@oneloop/box'
import { Icon } from '@oneloop/icons'
import { Text } from '@oneloop/text'
import { ButtonIcon } from '@oneloop/button'
import { Loader } from '@oneloop/loader'

export const Toast = ({ variant = 'base', variantLoader, variantText = 'body.fontSize14', icon, text, textDownload, download, ...props }) => (
  <Flex
    tx='toast'
    variant={variant}
    {...props}
    __css={{
      bg: 'white',
      width: '415px',
      height: '72px',
      boxShadow: 'active',
      borderRadius: '6px',
      backgroundColor: '#272E33',
      color: 'white',
      p: 1,
    }}
  >
    { icon !== undefined && (
    <Icon icon={icon} fontSize='29.33px' margin = '19px 15px'/>
    )}

    {variantLoader !== undefined && (
      <Loader variant={variantLoader} margin='19px 15px'/>
    )}

      {text !== undefined &&
      (<Text variant={variantText} display="flex" alignItems="center">
        {text}
      </Text>)}

      {download !== undefined && (
      <ButtonIcon icon='icon-descargar' variant='mainItemSmall' margin='20px 15px 20px 40px' fontSize='14px' color='white' text={textDownload} onClick={download}/>
      )}

    <Box margin='auto'>
      <ButtonIcon
        color='white'
        icon='icon-cerrar'
        variant={['subtleTrasnparentIconClearButton', 'iconLarge']}
        onClick={() => props.closeFunction()}/>
    </Box>
  </Flex>
)
