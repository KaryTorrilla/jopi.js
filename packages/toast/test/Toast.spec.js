import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

import { ButtonIcon } from '@oneloop/button'
import { Toast } from '../src'

Enzyme.configure({ adapter: new Adapter() })

describe('Toast', () => {
  test('renders correctly', () => {
    const tree = renderer.create(<Toast variant='base' text="Seleccione una propiedad" />).toJSON()

    expect(tree).toMatchSnapshot()
  })

  test('renders correctly three', () => {
    const tree = renderer.create(<Toast icon={'icon-alertas'} variant="alert" text="Alert message" />).toJSON()

    expect(tree).toMatchSnapshot()
  })

  test('renders correctly four', () => {
    const tree = renderer.create(<Toast variant="check" text="Success message" icon={'icon-check'} />).toJSON()

    expect(tree).toMatchSnapshot()
  })

  test('renders correctly four', () => {
    const tree = renderer.create(<Toast variant="error" text="Error message" icon={'icon-error'} />).toJSON()

    expect(tree).toMatchSnapshot()
  })

  test('renders correctly five', () => {
    const tree = renderer.create(<Toast variant="loading" text="Cargando..." variantLoader={'spinner'} />).toJSON()

    expect(tree).toMatchSnapshot()
  })

  test('renders correctly six', () => {
    const tree = renderer.create(<Toast variant="loading" text="Exportando..." variantLoader={'spinner'} />).toJSON()

    expect(tree).toMatchSnapshot()
  })

  test('renders correctly seven', () => {
    const tree = renderer.create(<Toast variant="success" text="Report export ready!" textDownload={'Download'} icon='icon-check' variantText='bodyBold.fontSize14' download={() => { console.log('hola, soy un texto que se le pasa al onclick para descargar') }}/>).toJSON()

    expect(tree).toMatchSnapshot()
  })

  test('closed', () => {
    const closeFuction = () => {}
    const component = shallow(<Toast variant="loading" text="Exportando..." variantLoader={'spinner'} closeFunction={closeFuction}/>)
    expect(component.find(<ButtonIcon/>).exists).toBeTruthy()
    component.find(ButtonIcon).simulate('click')

    expect(component).toMatchSnapshot()
  })
})
