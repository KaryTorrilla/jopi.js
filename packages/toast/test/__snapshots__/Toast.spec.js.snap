// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toast renders correctly 1`] = `
.c0 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  background-color: #272E33;
  width: 415px;
  height: 72px;
  box-shadow: active;
  border-radius: 6px;
  color: white;
  padding: 4px;
}

.c2 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  position: relative;
  width: 29.33px;
  height: 29.33px;
  margin: 20px 15px;
}

.c3 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  font-family: primary;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.c4 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  margin: auto;
}

.c5 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  position: relative;
}

.c6 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  line-height: inherit;
  font-family: Nunito Sans;
  font-weight: bold;
  text-align: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  cursor: pointer;
  outline: none;
  border: 0;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  white-space: nowrap;
  position: relative;
  color: white;
}

.c7 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  position: relative;
  width: 16px;
  height: 16px;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

@media screen and (min-width:40em) {

}

<div
  className="c0 c1"
>
  <div
    className="c2"
  >
    <i
      className="icon-check"
      style={
        Object {
          "bottom": "0px",
          "fontSize": "29.33px",
          "height": "29.33px",
          "left": "-0.5px",
          "position": "absolute",
          "right": "0px",
          "top": "-1px",
          "width": "29.33px",
        }
      }
    />
  </div>
  <p
    className="c3"
    display="flex"
  >
    Seleccione una propiedad
  </p>
  <div
    className="c4"
  >
    <div
      className="c5"
    >
      <button
        className="c6"
        color="white"
        onClick={[Function]}
      >
        <div
          className="c7"
        >
          <i
            className="icon-cerrar"
            style={
              Object {
                "bottom": "0px",
                "fontSize": "16px",
                "height": "16px",
                "left": "-0.5px",
                "position": "absolute",
                "right": "0px",
                "top": "-1px",
                "width": "16px",
              }
            }
          />
        </div>
      </button>
    </div>
  </div>
</div>
`;

exports[`Toast renders correctly five 1`] = `
.c0 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  background-color: #272E33;
  width: 415px;
  height: 72px;
  box-shadow: active;
  border-radius: 6px;
  color: white;
  padding: 4px;
}

.c2 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
}

.c3 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  position: relative;
  width: 29.33px;
  height: 29.33px;
  margin: 20px 15px;
  color: #00D186;
}

.c4 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  font-family: primary;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  font-weight: bold;
}

.c5 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  position: relative;
}

.c6 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  line-height: inherit;
  font-family: Nunito Sans;
  font-weight: normal;
  text-align: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  cursor: pointer;
  outline: none;
  border: 0;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  white-space: nowrap;
  position: relative;
  margin: 20px 15px 20px 40px;
  color: white;
  font-size: 14px;
}

.c7 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  position: relative;
  width: 12px;
  height: 12px;
}

.c8 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  margin: auto;
}

.c9 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  line-height: inherit;
  font-family: Nunito Sans;
  font-weight: bold;
  text-align: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  cursor: pointer;
  outline: none;
  border: 0;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  white-space: nowrap;
  position: relative;
  color: white;
}

.c10 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  position: relative;
  width: 16px;
  height: 16px;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

@media screen and (min-width:40em) {

}

<div
  className="c0 c1"
>
  <div
    className="c2"
    display="flex"
  >
    <div
      className="c3"
      color="#00D186"
    >
      <i
        className="icon-check"
        style={
          Object {
            "bottom": "0px",
            "fontSize": "29.33px",
            "height": "29.33px",
            "left": "-0.5px",
            "position": "absolute",
            "right": "0px",
            "top": "-1px",
            "width": "29.33px",
          }
        }
      />
    </div>
    <p
      className="c4"
      display="flex"
      fontWeight="bold"
    >
      Report export ready!
    </p>
    <div
      className="c5"
    >
      <button
        className="c6"
        color="white"
        fontSize="14px"
        onClick={[Function]}
      >
        <div
          className="c7"
        >
          <i
            className="icon-descargar"
            style={
              Object {
                "bottom": "0px",
                "fontSize": "12px",
                "height": "12px",
                "left": "-0.5px",
                "position": "absolute",
                "right": "0px",
                "top": "-1px",
                "width": "12px",
              }
            }
          />
        </div>
        <span>
          Descargar
        </span>
      </button>
    </div>
  </div>
  <div
    className="c8"
  >
    <div
      className="c5"
    >
      <button
        className="c9"
        color="white"
        onClick={[Function]}
      >
        <div
          className="c10"
        >
          <i
            className="icon-cerrar"
            style={
              Object {
                "bottom": "0px",
                "fontSize": "16px",
                "height": "16px",
                "left": "-0.5px",
                "position": "absolute",
                "right": "0px",
                "top": "-1px",
                "width": "16px",
              }
            }
          />
        </div>
      </button>
    </div>
  </div>
</div>
`;

exports[`Toast renders correctly four 1`] = `
.c0 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  background-color: #272E33;
  width: 415px;
  height: 72px;
  box-shadow: active;
  border-radius: 6px;
  color: white;
  padding: 4px;
}

.c2 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
}

.c3 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  margin-right: 10px;
  margin: 17px 15px;
  width: small;
  height: small;
}

.c5 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  font-family: primary;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.c6 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  margin: auto;
}

.c7 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  position: relative;
}

.c8 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  line-height: inherit;
  font-family: Nunito Sans;
  font-weight: bold;
  text-align: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  cursor: pointer;
  outline: none;
  border: 0;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  white-space: nowrap;
  position: relative;
  color: white;
}

.c9 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  position: relative;
  width: 16px;
  height: 16px;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c4 {
  width: 2em;
  height: 2em;
  padding: 8px;
  border-radius: 50%;
  background-image: conic-gradient(#272E33,#FFFFFF);
  -webkit-animation: bCrGrT 1s ease 0s infinite;
  animation: bCrGrT 1s ease 0s infinite;
  -webkit-clip-path: circle(80px at center);
  clip-path: circle(80px at center);
  -webkit-mask-image: radial-gradient(circle at center,transparent 54%,black 54.8%);
  -webkit-mask-image: radial-gradient(circle at center,transparent 54%,black 54.8%);
  mask-image: radial-gradient(circle at center,transparent 54%,black 54.8%);
}

@media screen and (min-width:40em) {

}

<div
  className="c0 c1"
>
  <loader
    className="c2"
  >
    <div
      className="c3 c4"
      kind="primary"
      size="small"
    />
  </loader>
  <p
    className="c5"
    display="flex"
  >
    Cargando...
  </p>
  <div
    className="c6"
  >
    <div
      className="c7"
    >
      <button
        className="c8"
        color="white"
        onClick={[Function]}
      >
        <div
          className="c9"
        >
          <i
            className="icon-cerrar"
            style={
              Object {
                "bottom": "0px",
                "fontSize": "16px",
                "height": "16px",
                "left": "-0.5px",
                "position": "absolute",
                "right": "0px",
                "top": "-1px",
                "width": "16px",
              }
            }
          />
        </div>
      </button>
    </div>
  </div>
</div>
`;

exports[`Toast renders correctly four 2`] = `
.c0 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  background-color: #272E33;
  width: 415px;
  height: 72px;
  box-shadow: active;
  border-radius: 6px;
  color: white;
  padding: 4px;
}

.c2 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
}

.c3 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  margin-right: 10px;
  margin: 17px 15px;
  width: small;
  height: small;
}

.c5 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  font-family: primary;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.c6 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  margin: auto;
}

.c7 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  position: relative;
}

.c8 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  line-height: inherit;
  font-family: Nunito Sans;
  font-weight: bold;
  text-align: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  cursor: pointer;
  outline: none;
  border: 0;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  white-space: nowrap;
  position: relative;
  color: white;
}

.c9 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  position: relative;
  width: 16px;
  height: 16px;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c4 {
  width: 2em;
  height: 2em;
  padding: 8px;
  border-radius: 50%;
  background-image: conic-gradient(#272E33,#FFFFFF);
  -webkit-animation: bCrGrT 1s ease 0s infinite;
  animation: bCrGrT 1s ease 0s infinite;
  -webkit-clip-path: circle(80px at center);
  clip-path: circle(80px at center);
  -webkit-mask-image: radial-gradient(circle at center,transparent 54%,black 54.8%);
  -webkit-mask-image: radial-gradient(circle at center,transparent 54%,black 54.8%);
  mask-image: radial-gradient(circle at center,transparent 54%,black 54.8%);
}

@media screen and (min-width:40em) {

}

<div
  className="c0 c1"
>
  <loader
    className="c2"
  >
    <div
      className="c3 c4"
      kind="primary"
      size="small"
    />
  </loader>
  <p
    className="c5"
    display="flex"
  >
    Exportando...
  </p>
  <div
    className="c6"
  >
    <div
      className="c7"
    >
      <button
        className="c8"
        color="white"
        onClick={[Function]}
      >
        <div
          className="c9"
        >
          <i
            className="icon-cerrar"
            style={
              Object {
                "bottom": "0px",
                "fontSize": "16px",
                "height": "16px",
                "left": "-0.5px",
                "position": "absolute",
                "right": "0px",
                "top": "-1px",
                "width": "16px",
              }
            }
          />
        </div>
      </button>
    </div>
  </div>
</div>
`;

exports[`Toast renders correctly three 1`] = `
.c0 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  background-color: #272E33;
  width: 415px;
  height: 72px;
  box-shadow: active;
  border-radius: 6px;
  color: white;
  padding: 4px;
}

.c2 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  position: relative;
  width: 29.33px;
  height: 29.33px;
  margin: 18px 15px;
  color: #00D186;
}

.c3 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  font-family: primary;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.c4 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  margin: auto;
}

.c5 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  position: relative;
}

.c6 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  line-height: inherit;
  font-family: Nunito Sans;
  font-weight: bold;
  text-align: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  cursor: pointer;
  outline: none;
  border: 0;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  white-space: nowrap;
  position: relative;
  color: white;
}

.c7 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  position: relative;
  width: 16px;
  height: 16px;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

@media screen and (min-width:40em) {

}

<div
  className="c0 c1"
>
  <div
    className="c2"
    color="#00D186"
  >
    <i
      className="icon-check"
      style={
        Object {
          "bottom": "0px",
          "fontSize": "29.33px",
          "height": "29.33px",
          "left": "-0.5px",
          "position": "absolute",
          "right": "0px",
          "top": "-1px",
          "width": "29.33px",
        }
      }
    />
  </div>
  <p
    className="c3"
    display="flex"
  >
    Success message
  </p>
  <div
    className="c4"
  >
    <div
      className="c5"
    >
      <button
        className="c6"
        color="white"
        onClick={[Function]}
      >
        <div
          className="c7"
        >
          <i
            className="icon-cerrar"
            style={
              Object {
                "bottom": "0px",
                "fontSize": "16px",
                "height": "16px",
                "left": "-0.5px",
                "position": "absolute",
                "right": "0px",
                "top": "-1px",
                "width": "16px",
              }
            }
          />
        </div>
      </button>
    </div>
  </div>
</div>
`;

exports[`Toast renders correctly two 1`] = `
.c0 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  background-color: #272E33;
  width: 415px;
  height: 72px;
  box-shadow: active;
  border-radius: 6px;
  color: white;
  padding: 4px;
}

.c2 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  position: relative;
  width: 29.33px;
  height: 29.33px;
  margin: 20px 15px;
  color: #A2B5F7;
}

.c3 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  font-family: primary;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.c4 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  margin: auto;
}

.c5 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  position: relative;
}

.c6 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  line-height: inherit;
  font-family: Nunito Sans;
  font-weight: bold;
  text-align: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  cursor: pointer;
  outline: none;
  border: 0;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  white-space: nowrap;
  position: relative;
  color: white;
}

.c7 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  position: relative;
  width: 16px;
  height: 16px;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

@media screen and (min-width:40em) {

}

<div
  className="c0 c1"
>
  <div
    className="c2"
    color="#A2B5F7"
  >
    <i
      className="icon-alertas"
      style={
        Object {
          "bottom": "0px",
          "fontSize": "29.33px",
          "height": "29.33px",
          "left": "-0.5px",
          "position": "absolute",
          "right": "0px",
          "top": "-1px",
          "width": "29.33px",
        }
      }
    />
  </div>
  <p
    className="c3"
    display="flex"
  >
    Alert message
  </p>
  <div
    className="c4"
  >
    <div
      className="c5"
    >
      <button
        className="c6"
        color="white"
        onClick={[Function]}
      >
        <div
          className="c7"
        >
          <i
            className="icon-cerrar"
            style={
              Object {
                "bottom": "0px",
                "fontSize": "16px",
                "height": "16px",
                "left": "-0.5px",
                "position": "absolute",
                "right": "0px",
                "top": "-1px",
                "width": "16px",
              }
            }
          />
        </div>
      </button>
    </div>
  </div>
</div>
`;
